lib/spack/llnl/util/filesystem.py

```diff
@@ -85,6 +85,13 @@
 ]
 
 
+def remove_file_mount(path):
```

Comment:
I don't like the name of this function since it does more than it sounds like. The function changes C:\\.....\\Temp\\...\\repo.git into file\\Temp\\...\\repo.git

Reply:
It removes everything before Temp due to long path issues with git clone